tracer.go

```diff
@@ -145,6 +145,11 @@ type StartSpanOptions struct {
 	// If specified, the caller hands off ownership of Tags at
 	// StartSpan() invocation time.
 	Tags map[string]interface{}
+
+	// Perfevent is the platform metric's name(s) or an event name
+	// supported by perfevents package
+	// (https://github.com/opentracing-contrib/perfevents/go)
+	Perfevent string
```

Comment:
Hi @yurishkuro, thanks for the review.
- Sure, will change it to PerfEvent
- Yeah. The comments should have been clearer. PerfEvent is a list of perf events in the form of a comma separated string (e.g., "cache-misses,cpu-cycles,instructions"). The tracer is expected to start the events in PerfEvent using the "perfevents" package (https://github.com/opentracing-contrib/perfevents/go) . On span Finish(), the events must be closed. The domain of the accepted values is the set of events supported by the "perfevents" package. Having it as an enum may pose difficulties since, we are also planning to support some dynamic events and other metrics taken from the kernel, in future. Hence, having it as an enum may pose challenges.
- I agree. Let me check this out and if it proves to be sufficient, will update this PR.